### PR TITLE
[skipci] feat: 增加curvebs nfs 共享存储实践 

### PR DESCRIPTION
**Pull request by DemoLiang**
Signed-off-by: DemoLiang 

### What problem does this PR solve?

Issue Number:  none

Problem Summary: 
通过curvebs 部署块存储测试集群
通过nfs-server部署nfs服务，提供可挂载的nfs服务
通过nfs挂载提供curvebs的共享存储使用

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/practical/curvebs_nfs.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+
+### 环境信息
+
+主机名|系统|IP|用户名密码
+-|-|-|-
+主控机|centos7|172.20.0.62|root / XX....
+server-node1|centos7|172.20.0.38|root / vT2dALVE
+server-node2|centos7|172.20.0.39|root / W0lFAv1h
+server-node3|centos7|172.20.0.58|root / YVcR8Iwl
+client-node1|centos7|172.20.0.61| root / KR0c61Vs
```
Comment (Cyber-SiKu) [inline docs/practical/curvebs_nfs.md:12]: algin

File: docs/practical/curvebs_nfs.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+```shell
+curveadm client status 
+Get Client Status: [OK]  
+
+Id            Kind     Host          Container Id  Status                           Aux Info                                          
+--            ----     ----          ------------  ------                           --------                                          
+c7b6d6ce8a77  curvebs  client-node1  7a35f31c5648  Exited (143) About an hour ago   {"user":"curvebs_user","volume":"/dev/curvebs_bs"}
+f1225dfb6b94  curvebs  client-node1  4a81d6b2ac6f  Exited (143) About a minute ago  {"user":"curvebs_user","volume":"/vdq"}           
+4d01b8991a8a  curvebs  client-node1  5008c7362d66  Exited (143) 25 minutes ago      {"user":"curvebsuser1","volume":"/vdq"}           
+f846805e752e  curvebs  client-node1  c0c0ceb45459  Exited (143) 25 minutes ago      {"user":"curvebs_user","volume":"/vdw"}
```
Comment (Cyber-SiKu) [inline docs/practical/curvebs_nfs.md:263]: remove extra whitespace

File: docs/practical/curvebs_nfs.md (hunk 1 above)
Comment (Cyber-SiKu) [inline docs/practical/curvebs_nfs.md:12]: algin

**Comment by Cyber-SiKu** [comment]
You can add "skipci" to the title to avoid ci.

File: docs/practical/curvebs_nfs.md (hunk 1 above)
Comment (DemoLiang) [reply docs/practical/curvebs_nfs.md]: update

**Review by zhanghuidinah — COMMENTED** [review]
Hi @DemoLiang thanks for your contribution! 🎉 It is recommended to encrypt IP and password.

**Review by Cyber-SiKu — APPROVED** [review]

**Review by opencurveadmin — APPROVED** [review]

**Pull request merged**